cli/components/distributions.go

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package components
+
+import (
+	"fmt"
+
+	"github.com/charmbracelet/bubbles/table"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+	"go.mondoo.com/cnquery/v11/providers-sdk/v1/inventory"
+	"go.mondoo.com/cnspec/v11/policy"
+)
+
+var (
+	// Custom table title style.
+	titleStyle = lipgloss.NewStyle().
+			Bold(false).
+			MarginBottom(0)
+
+	// Custom table cell style.
+	cellStyle = lipgloss.NewStyle().Padding(0, 0)
+)
+
+// DistributionsModel renders two tables, scores and asset distribution tables.
+//
+// E.g.
+//
+// Score Distribution                        Asset Distribution
+// -------------------------------------     ----------------------------------------
+// CRITICAL             1 assets             Ubuntu 16.04.7 LTS             1
+// HIGH                 1 assets
+// MEDIUM               1 assets
+// LOW                  1 assets
```

Comment:
nit pick and personal preference:
To me, these tables could be a bit more narrow.